### PR TITLE
draft web auth additions

### PR DESCRIPTION
**Pull request by d45**
@illonage @alexeski @Stephen-Price

I updated the "non-view" page in the playbook with some basic embedded web authoring information. Perhaps too much detail.  Let me know what's missing and what we should cover. And feel free to add other reviewers. 

There are a couple sections at the end that I'm not sure still apply (under "Additional Considerations" and "Embedding Other Tableau Server Pages").

Thanks, Dave

### CODE REVIEW

File: pages/06_embedding_non_view_content.md [hunk 1]
```diff
@@ -2,22 +2,64 @@
 title: Embedding Non-View Content
 ---
 
-Almost all embedded deployments of Tableau embed pre-built dashboards. But many embed other components of Tableau Server such as:
+Almost all embedded deployments of Tableau embed pre-built dashboards. But some embed other components of Tableau such as:
 
-* Web Authoring screens
+* web authoring views
```
Comment (illonage) [inline pages/06_embedding_non_view_content.md]: Web Authoring experience

File: pages/06_embedding_non_view_content.md [hunk 2]
```diff
@@ -2,22 +2,64 @@
...
 * The Tableau Server UI (embedded for integration and single sign-on reasons)
```
Comment (illonage) [inline pages/06_embedding_non_view_content.md]: We should add: "Not supported by Tableau"

File: pages/06_embedding_non_view_content.md [hunk 3]
```diff
@@ -2,22 +2,64 @@
...
 
-There are no APIs to achieve this type of embedding, but you can use iframes and any of the [single sign-on capabilities]({{ site.baseurl }}/pages/02_auth_and_sso) to achieve this.
+The easiest and best way to include web authoring in your embedding scenario is to use the [Tableau Embedding API v3](https://help.tableau.com/current/api/embedding_api/en-us/index.html). The Tableau Embedding API v3 provides a web component, `<tableau-viz-authoring>`, which in just a few lines of HTML code, allows users to author the view in your web application. You can also use the Embedding API v3 to call JavaScript methods that let you dynamically configure and switch between the authoring or viewing experience.
```
Comment (illonage) [inline pages/06_embedding_non_view_content.md]: "allows users to author the view in your web application" => We should add if permissions allowed. 
"You can also use the Embedding API v3 to call JavaScript methods that let you dynamically configure and switch between the authoring or viewing experience" => Allow you to customize authoring lifecycle, for example you can dynamically configure and switch between the authoring or viewing experience

File: pages/06_embedding_non_view_content.md [hunk 4]
```diff
@@ -2,22 +2,64 @@
...
 
 ## Embedding Web Authoring
 
-Web Authoring is the Tableau Desktop-like functionality in the browser. The fact that Web Authoring exists allows you to pass on the self-service capabilities of Tableau to your end users. You may have pre-built dashboards for them to use, but they may have minor customization request or may want to build visualizations and dashboards from scratch.
+Web authoring allows someone to edit a visualization just as they would in Tableau Desktop.
+Embedding a view with web authoring gives your users the ability to modify and update a view, all while staying in their workflow. Embedded web authoring means that you can create comprehensive embedded applications for your customers that now include options for self-servicing.
```
Comment (illonage) [inline pages/06_embedding_non_view_content.md]: remove "now"

File: pages/06_embedding_non_view_content.md [hunk 5]
```diff
@@ -2,22 +2,64 @@
...
 
-For many situations, it is acceptable to simply allow the default behavior of allowing the user to click 'Edit' on the toolbar of an embedded dashboard. The result will be a new browser tab opened to the web authoring screen for that workbook. However, if you want to keep the user inside your application, you may prefer to embed the web authoring capabilities. To do that:
+When it comes to editing a visualization, it is often acceptable to simply use the default settings and allow the user to click **Edit** on the toolbar of an embedded dashboard. The result will be a new browser tab opened to the web authoring screen for that workbook. However, if you want to keep the user inside your application, you might prefer to embed the web authoring capabilities. To do that:
```
Comment (illonage) [inline pages/06_embedding_non_view_content.md]: Now they can embed Tableau authoring experience directly from the application, no need to jump to another window and tab. They can use the button in the toolbar or create their own button that match their own brand

File: pages/06_embedding_non_view_content.md [hunk 6]
```diff
@@ -2,22 +2,64 @@
...
+
+1. In the `<tableau-authoring-viz>` component, add the `onWorkbookReadyToClose` event attribute to know when to switch back to view mode. In your custom event handler for the close event, hide the authoring component from view and show the viewing component.
+
+For an overview of the embedded web authoring feature, see [How to Enable Self-Service Analytics in Your Application with Embedded Web Authoring](https://www.tableau.com/about/blog/2022/8/how-enable-self-service-analytics-your-application-embedded-web-authoring). For a hands-on tutorial, see [Embedded API - Web Authoring Tutorial](https://www.tableau.com/developer/learning/embedding-api-web-authoring-tutorial). And for more information, see [Embedded Web Authoring](https://help.tableau.com/current/api/embedding_api/en-us/docs/embedding_api_web_authoring.html) in the Embedding API v3 Help documentation.
+
+
+<!-- Not sure if we need to keep any of the following. Does it still apply? -->
+// TO DO: Does the following still apply?
```
Comment (illonage) [inline pages/06_embedding_non_view_content.md]: one and two are still accurate. We should remove 3 and replace 4 by Connected Apps and be sure to pass the right scope

**Review by illonage — CHANGES_REQUESTED** [review]

File: pages/06_embedding_non_view_content.md (hunk 2 above)
Comment (d45) [reply pages/06_embedding_non_view_content.md]: @illonage  - you mean add the "Not supported by Tableau" to the second bullet item?  That is, the Tableau Server UI (SSO)?

* The Tableau Server UI (embedded for integration and single sign-on reasons). (Note that Tableau does not offer support for this deployment).

File: pages/06_embedding_non_view_content.md (hunk 1 above)
Comment (d45) [reply pages/06_embedding_non_view_content.md]: done

File: pages/06_embedding_non_view_content.md (hunk 3 above)
Comment (d45) [reply pages/06_embedding_non_view_content.md]: done.

File: pages/06_embedding_non_view_content.md (hunk 2 above)
Comment (illonage) [reply pages/06_embedding_non_view_content.md]: yes correct, just for bullet 2

File: pages/06_embedding_non_view_content.md (hunk 4 above)
Comment (d45) [reply pages/06_embedding_non_view_content.md]: done

File: pages/06_embedding_non_view_content.md (hunk 5 above)
Comment (d45) [reply pages/06_embedding_non_view_content.md]: Good suggestion. Updated paragraph: 

When it comes to editing a visualization, it is often acceptable to simply use the default settings and allow the user to click **Edit** on the toolbar of an embedded dashboard. The result will be a new browser tab opened to the web authoring screen for that workbook. However, now you can embed the Tableau authoring experience directly in the application. There's no need to have users jump to another window or tab. You can use the **Edit** button in the toolbar, or create your own edit button that matches the look and feel of your application. To do that:

File: pages/06_embedding_non_view_content.md (hunk 6 above)
Comment (d45) [reply pages/06_embedding_non_view_content.md]: Thanks! 

* If you are using Connected Apps for embedding views, be sure to set the scope (`scp`) claim in the Jason Web Token (JWT) to include web authoring (`tableau:views:embed_authoring`). For information about Connected Apps, see [Authentication and Single Sign-On (SSO)]({{site.baseurl}}/pages/02_auth_and_sso) and [Configure Tableau Connected Apps to Enable SSO for Embedded Content](https://help.tableau.com/current/online/en-us/connected_apps.htm).

**Review by illonage — APPROVED** [review]

File: pages/06_embedding_non_view_content.md (hunk 6 above)
Comment (d45) [reply pages/06_embedding_non_view_content.md]: yikes, Jason!  => JSON

**Pull request merged**